apps/ios/SimpleXChat/ChatTypes.swift

```diff
@@ -2112,6 +2112,17 @@ public struct ChatItem: Identifiable, Decodable {
         return nil
     }
 
+    public var encryptedFile: Bool? {
+        guard let fileSource = file?.fileSource else { return nil }
+        return fileSource.cryptoArgs != nil
+    }
+
+    public var encryptLocalFile: Bool {
+        file?.fileProtocol == .xftp &&
+        content.msgContent?.isVideo == false &&
```

Comment:
weren't regular files going to not be encrypted initially too?

Reply:
they are now encrypted - it's the same logic as with voice - they are decrypted to temp folder and removed after sharing activity completes (in the completion handler)

Reply:
it also optimized the logic of copying the file over to app storage - previously it was fully read into memory instead of just copying, for reasons unknown...

Reply:
imaging trying to send 1gb file